flac.md

```diff
@@ -987,6 +989,58 @@ Change controller: Internet Engineering Task Force
 Provisional registration? (standards tree only): NO
 ```
 
+## Application ID Registry
+
+This document creates a new IANA registry called the "FLAC Application Metadata Block ID" registry. The values correspond to the 32-bit identifier described in (#application).
+
+To register a new Application ID in this registry, one needs an Application ID, a description, optionally a reference to a document describing the Application ID and a Change Controller (IETF or email of registrant). The Application IDs are to be allocated according to the "First Come First Served" policy [RFC8126]. An Application ID can be any 32-bit value, but is often composed of 4 ASCII characters, to be human-readable.
+
+The FLAC Application Metadata Block ID registry is assigned the following initial values.
```

Comment:
@ktmf01, could you help me with a couple of things here? 

My understanding is that we want application IDs to be registered if they are in use, because FLAC audio files encoded using any of these application IDs can pop up at an arbitrary point in the future, and it's better for someone who needs to decode the audio file to know where to start looking - this is why FCFS is the correct choice for the registry policy. 

My understanding is that we are taking over a list of application IDs from https://xiph.org/flac/id.html, that hasn't been maintained, and many of the links on that page return 404s, etc, so although XIPH did have a reference for BUGS when it was registered, we can't provide a reference for BUGS for anyone who wants to decode BUGS audio files stored in FLAC is that correct?

Assuming so, for both of these, perhaps it's useful to say this as 

> The Application IDs are to be allocated according to the "First Come First Served" policy [RFC8126], so that there is no impediment to registering any Application IDs the FLAC community encounters, especially if they were used in audio files but were not registered when the audio files were encoded. 

and 

> The FLAC Application Metadata Block ID registry is assigned the following initial values, [taken from the XIPH.org registration page](https://xiph.org/flac/id.html).

Especially to @ktmf01 and @mcr - does that make sense?

Reply:
> My understanding is that we are taking over a list of application IDs from https://xiph.org/flac/id.html, that hasn't been maintained, and many of the links on that page return 404s, etc, so although XIPH did have a reference for BUGS when it was registered, we can't provide a reference for BUGS for anyone who wants to decode BUGS audio files stored in FLAC is that correct?

Not necessarily. Providing a specification has always been optional when registering an ID. So, yes, Xiph did have a reference for BUGS, but not a specification. I've pruned that list down to instances where some form of specification is known to me.

For example, the reference for BUGS is http://www.neowizbugs.com/ which doesn't exist anymore. However, I'm reasonably sure that this player is related to a Korean music streaming service with the same name that still exists: https://en.wikipedia.org/wiki/Bugs!_(streaming_service) I haven't been able to find a specification, so I've not included any link.

---

Considering your suggestion, I think it makes perfect sense, I'll include it. I'll make the link an informative reference instead of just a link however.